certbot/tests/crypto_util_test.py

```diff
@@ -236,6 +239,80 @@ def test_it(self):  # pylint: disable=no-self-use
         OpenSSL.crypto.load_privatekey(
             OpenSSL.crypto.FILETYPE_PEM, make_key(1024))
 
+class VerifyRenewableCertTest(unittest.TestCase):
```

Comment:
Rather than putting tests for all four functions in a single class, I think it's better practice to split them into a separate class for each function. Doing this breaks everything into smaller and more digestible pieces.

You can make a common base class for the pieces that are the same. There is an example of doing this in `certbot/tests/cert_manager_test.py`.